undebt/pattern/interface.py

```diff
@@ -74,10 +76,35 @@ def patterns_from_files(pattern_files):
 
 def load_module(path):
     """Loads a module from its path."""
+    if module_like(path):
+        return _load_module(path)
+
     pattern_name = os.path.splitext(os.path.basename(path))[0]
     return imp.load_source(pattern_name, path)
 
 
+def _load_module(full_name):
+    try:
+        return importlib.import_module(full_name)
+    except ImportError as e:
+        raise e
+    finally:
+        # Unshim path
+        sys.path = sys.path[1:]
```

Comment:
probably, but if it isn't during normal python execution I don't think undebt should go out of its way to attempt to make it importable.